.eslintrc.json

```diff
@@ -109,7 +110,15 @@
 			}
 		],
 		"space-infix-ops": "error",
-		"no-only-tests/no-only-tests": "error"
+		"no-only-tests/no-only-tests": "error",
+		"import/first": "error",
+		"import/named": "error",
+		"import/namespace": "error",
+		"import/default": "error",
+		"import/export": "error",
+		"import/order": "error",
+		"import/no-duplicates": "error",
+		"import/newline-after-import": "error"
```

Comment:
Can't we add this as `warning` instead of `error`?

Reply:
@hleote there's a plugin for that 😂 https://github.com/bfanger/eslint-plugin-only-warn